main.py

```diff
@@ -60,3 +60,52 @@ def infinite_loop():
         if len(w) > 4:
             words.insert(0, w)
     return words
+
+
+# Prefer DefaultDict over setDefult
+def setdefault_example():
+    std_dict = dict()
+    for k, v  in enumerate(range(5)):
+        std_dict.setdefault(k, []).append(v)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You are using the `dict.setdefault()` method. We recommend using the `defaultdict` object to handle missing items in internal state because the default value is automatically generated when you access or modify a missing key. `defaultdict` can also be easier to use and understand.

[Learn more](https://docs.python.org/3/library/collections.html#collections.defaultdict)